lxd/cluster/config/config.go

```diff
@@ -248,6 +248,23 @@ func (c *Config) OIDCServer() (string, string, string) {
 	return c.m.GetString("oidc.issuer"), c.m.GetString("oidc.client.id"), c.m.GetString("oidc.audience")
 }
 
+// ClusterHealing returns the configured healing threshold, i.e. the
+// number of seconds after which an offline node will be evacuated automatically. If the config key
+// is set but its value is lower than cluster.offline_threshold it returns
+// the value of cluster.offline_threshold instead.
+func (c *Config) ClusterHealing() time.Duration {
```

Comment:
In this function I think I would have got `cluster.healing` int value as you have done, then converted it to a time.Duration, before then getting the value of `c.OfflineThreshold()` and comparing them as time.Durations. That way I've not accessed `cluster.offline_threshold` in 2 different ways. The way it is now it caught me eye as a reviewer because I couldn't immediately understand why you've access it as an int first and then retrieved it again using `c.OfflineThreshold()` (Once I looked at the implementation of `OfflineThreshold` I realised that it was because it returns time.Duration).

Either that or use `c.m.GetInt64("cluster.offline_threshold")` and then convert that to time.Duration in this function instead of calling `OfflineThreshold`.